source/lib/robot_model/tests/testRobotModel.cpp

```diff
@@ -0,0 +1,16 @@
+#include <gtest/gtest.h>
+#include "robot_model/Model.hpp"
+
+using namespace RobotModel;
+
+TEST(LoadModel, PositiveNos)
+{
+	Model model("staubli", "/home/ros2/models/staubli_rx160/staubli_rx160.urdf");
```

Comment:
I know testing is off but should this be removed or changed if we dont include the urdfs here?

Reply:
yes that is a fair point. Either we remove it or include a test model somehow